nvd3/multiChart.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Python-nvd3 is a Python wrapper for NVD3 graph library.
+NVD3 is an attempt to build re-usable charts and chart components
+for d3.js without taking away the power that d3.js gives you.
+
+Project location : https://github.com/areski/python-nvd3
+"""
+
+from .NVD3Chart import NVD3Chart, TemplateMixin
+
+
+class multiChart(TemplateMixin, NVD3Chart):
+
+    """
+    A line chart or line graph is a type of chart which displays information
+    as a series of data points connected by straight line segments.
+
+    Python example::
+
+        from nvd3 import lineChart
```

Comment:
The inlines docs needs to be updated to reflect multiChat also the example and the output